concourse/pipelines/deploy.yml

```
@@ -183,23 +142,68 @@ resources:
       initial_version: "0"
 
 
-  - name: frontend-image
-    type: registry-image
+  - &registry-image
+    name: frontend-image
+    type: registy-image-v2
     icon: docker
+    source: &registry-source
+      repository: frontend
+      aws_role_arn: arn:aws:iam::172025368201:role/pull_images_from_ecr_role
+      aws_region: eu-west-1
+      aws_access_key_id: ((concourse-ecr-readonly-user_aws-access-key))
+      aws_secret_access_key: ((concourse-ecr-readyonly-user_aws-secret-access-key))
```

typo - check if the actual secret name needs to be fixed too
```suggestion
      aws_secret_access_key: ((concourse-ecr-readonly-user_aws-secret-access-key))
```

---

will fix, need secret fix too but easy to do